requirements.txt

```diff
@@ -37,6 +40,8 @@ MarkupSafe==1.1.1
 mccabe==0.6.1
 mock==2.0.0
 more-itertools==6.0.0
+mypy==0.670
```

Comment:
Do we need the libraries `mypy`, `mypy-extensions`, `typed-ast` and `typing-extensions` for this feature fix?